src/main/resources/hibernate.cfg.xml

```diff
@@ -15,6 +15,9 @@
         <property name="hbm2ddl.auto">create-drop</property>
 
         <mapping class="core.basesyntax.model.zoo.Animal"/>
+        <mapping class="core.basesyntax.model.figure.Figure"/>
```

Comment:
Figure is entity?
Still there are other classes

Reply:
nope